packages/back-end/src/services/experimentNotifications.ts

```diff
@@ -418,8 +418,8 @@ export const notifyExperimentChange = async ({
   const experiment = await getExperimentById(context, snapshot.experiment);
   if (!experiment) throw new Error("Error while fetching experiment!");
 
-  // do not fire significance or error events for dimension analyses
-  if (snapshot.dimension) return;
+  // do not fire significance or error events for ad-hoc analyses
+  if ((snapshot.type ?? "manual") === "ad-hoc") return;
```

Comment:
If `snapshot.type` is undefined, it means it was stored in Mongo before this PR and may have a dimension.  It's safer to just make this return if it's undefined.